packages/button/theme.module.css

```diff
@@ -132,40 +132,31 @@
   }
 
   &.brand-neutral {
-    &.background-dark {
+    --background-color: transparent;
+    --border-color-hovered: transparent;
+    --border-color: transparent;
+    --padding-medium: 0;
+    --padding-small: 0;
+    --scrim-opacity-active: 0;
+    --scrim-opacity-hovered: 0;
+    --text-color: var(--black);
+
+    &.background-dark,
+    &.background-brand {
       --text-color: var(--white);
     }
-  }
 
-  &[disabled] {
-    &.background-light,
-    &.background-brand-secondary {
-      --text-color: rgba(0, 0, 0, 0.3);
-    }
-    &.background-brand,
-    &.background-dark {
-      --text-color: rgba(255, 255, 255, 0.3);
+    &[disabled] {
+      &.background-light,
+      &.background-brand-secondary {
+        --text-color: rgba(0, 0, 0, 0.3);
+      }
+      &.background-brand,
+      &.background-dark {
+        --text-color: rgba(255, 255, 255, 0.3);
+      }
     }
   }
-}
-
-/*
- * Tertiary neutral variant
- */
-.variant-tertiary-neutral {
```

Comment:
I think this `.variant-tertiary-neutral` is still necessary to render that variant. `*`

I think the fix here might be simpler - eg [something like this](https://github.com/hashicorp/react-components/pull/295/commits/b552d23818c1adb6cf73dd383b25a8a0a251cd4d) (cherry-pick `b552d23 ` if you want) seems to produce [the expected sticker sheet appearance](https://react-components-storybook.vercel.app/?path=/story/components-button--themes).

---

`*` To be clear, `tertiary-neutral` is very duplicative of `{ variant: "tertiary", brand: "neutral" }`, it stemmed a design request to be able to conveniently ignore the brand colour in certain situations, as increasingly design wanted "neutral"-brand tertiary buttons alongside product-branded primary buttons. It was probably not the best thing to do, feels like it adds more confusion for the benefit of a very tiny amount of convenience.

Reply:
Oh shoot yeah I did not realize that one could use `tertiary-neutral` as the variant. I can add that back 👀. Thanks!